apps/api/src/app/integrations/usecases/create-integration/create-integration.usecase.ts

```diff
@@ -80,6 +82,13 @@ export class CreateIntegration {
       throw new BadRequestException('One environment can only have one In app provider');
     }
 
+    if (
+      (command.providerId === SmsProviderIdEnum.Novu && !areNovuSmsCredentialsSet()) ||
+      (command.providerId === EmailProviderIdEnum.Novu && !areNovuEmailCredentialsSet())
+    ) {
+      throw new BadRequestException(`Creating Novu integration for ${command.providerId} provider is not allowed`);
+    }
```

Comment:
when creating the Novu integration for SMS or Email check if the environment variables for the provider credentials are set, if not throw an exception